pkg/crc/cluster/cluster.go

```diff
@@ -236,7 +236,7 @@ func AddProxyToKubeletAndCriO(sshRunner *ssh.Runner, proxy *network.ProxyConfig)
 	proxyTemplate := `[Service]
 Environment=HTTP_PROXY=%s
 Environment=HTTPS_PROXY=%s
-Environment=NO_PROXY=.cluster.local,.svc,10.128.0.0/14,172.30.0.0/16,%s`
+Environment=NO_PROXY=.cluster.local,.svc,10.116.0.0/14,172.25.0.0/16,192.168.126.0/24,%s`
```

Comment:
Can you add in the commit log this is because of https://github.com/code-ready/snc/blob/master/install-config.yaml file, it took me some time to get that info.

Reply:
oh great! I didnt found it yesterday. I just compared what was injected automatically and what we hardcoded here.
Thanks.